### ISSUE TITLE
Failed to add face_geometry to output stream, can't find file "geometry_pipeline_metadata.binarypb"

### ISSUE THREAD

**Issue by xiao668** [google/mediapipe#1253]
Hi, I'm trying to output the pose transformation matrix in desktop samples, like the sample **FaceEffectApp** in iOS. I got to know from the documentation that I need to add the geometry pipeline to the sample code in order to get the pose matrix. Below is what I did:

1. In the "demo_run_graph_main.cc", I included the headers:
```
#include "mediapipe/framework/formats/matrix_data.pb.h"
#include "mediapipe/modules/face_geometry/protos/face_geometry.pb.h"
``` 

Then, I added an output stream poller to listen for output face geometry packets:
`ASSIGN_OR_RETURN(::mediapipe::OutputStreamPoller output_geometry_poller, graph.AddOutputStreamPoller(kMultiFaceGeometryStream));`

At each frame, check if a packet is available in the output stream and output the matrix information:
 ```
     ::mediapipe::Packet multi_face_geometry_packet;
      if (!output_geometry_poller.Next(&multi_face_geometry_packet)) {
        return ::mediapipe::UnknownError(
            "Failed to get packet from output stream 'multi_face_geometry'.");
      }
      const auto& multi_face_geometry = multi_face_geometry_packet.Get<std::vector<::mediapipe::face_geometry::FaceGeometry>>();
      LOG(INFO) << "Number of face instances with landmarks: " << multi_face_geometry.size();
      for (int face_index = 0; face_index < multi_face_geometry.size(); ++face_index) {
          const auto& face_geometry = multi_face_geometry[face_index];
          LOG(INFO) << "Approx. distance away from camera for face" << face_index << ": " << face_geometry.pose_transform_matrix().packed_data(14) << " cm.";
      }
```

2. In "face_mesh_desktop_live.pbtxt":

I added an additional output_stream: `output_stream: "multi_face_geometry"`

Then, generate an environment that describes the current virtual scene.
```
 node {
   calculator: "FaceGeometryEnvGeneratorCalculator"
   output_side_packet: "ENVIRONMENT:environment"
   node_options: {
     [type.googleapis.com/mediapipe.FaceGeometryEnvGeneratorCalculatorOptions] {
       environment: {
         origin_point_location: TOP_LEFT_CORNER
         perspective_camera: {
           vertical_fov_degrees: 63.0  # 63 degrees
           near: 1.0  # 1cm
           far: 10000.0  # 100m
         }
       }
     }
   }
 }
```

And extract the throttled input image frame dimensions as a separate packet.
```
node {
    calculator: "ImagePropertiesCalculator"
    input_stream: "IMAGE:throttled_input_video"
    output_stream: "SIZE:input_video_size"
}
```

subgraph that computes face geometry from landmarks for a single face.
```
node {
     calculator: "FaceGeometry"
     input_stream: "MULTI_FACE_LANDMARKS:multi_face_landmarks"
     input_stream: "IMAGE_SIZE:input_video_size"
     input_side_packet: "ENVIRONMENT:environment"
     output_stream: "MULTI_FACE_GEOMETRY:multi_face_geometry"
}
```

The code can build successfully, however, during run time, it says "**Can't find file: mediapipe/modules/face_geometry/data/geometry_pipeline_metadata.binarypb**", as shown below:
```
I20201104 21:54:17.467227 253103552 demo_run_graph_main.cc:63] Initialize the calculator graph.
I20201104 21:54:18.444079 253103552 demo_run_graph_main.cc:81] Set up live cam window size.
I20201104 21:54:18.572449 253103552 demo_run_graph_main.cc:90] Start running the calculator graph.
I20201104 21:54:18.572783 253103552 demo_run_graph_main.cc:104] Start grabbing and processing frames.
I20201104 21:54:19.573032 253103552 demo_run_graph_main.cc:206] Shutting down.
E20201104 21:54:19.816355 253103552 demo_run_graph_main.cc:217] Failed to run the graph: CalculatorGraph::Run() failed in Run: 
Calculator::Open() for node "facegeometry__FaceGeometryPipelineCalculator" failed: ; Can't find file: mediapipe/modules/face_geometry/data/geometry_pipeline_metadata.binarypb; Failed to read content blob! Resolved path = mediapipe/modules/face_geometry/data/geometry_pipeline_metadata.binarypb; Failed to read a metadata blob from file!; Failed to read the geometry pipeline metadata from file!
```

Do I need to modify the BUILD file somewhere to get the geometry_pipeline_metadata.binarypb built at runtime execution? If so, which path/deps should I include and in which BUILD file?

**Comment by konovalov-k** [comment]
For my custom build for Android, I put this geometry_pipeline_metadata.binarypb from bazel-out (after successfull build .aar for face_effect) to assets folder. Maybe you must do something similar.

**Comment by ianni67** [comment]
I'm experiencing the same issue. On a desktop (linux) the file geometry_pipeline_metadata.binarypb is not built.
What can I do?

**Comment by ianni67** [comment]
The issue is hopefully resolved in #1584

**Comment by kostyaby** [comment]
Hey folks,

Sorry for missing this issue all way back in November! I left some reasoning & recommendation as a comment inside #1584 - also, consider using @ianni67's method for finding the output path described inside #1584 as well
